config/desktop-development.json

```diff
@@ -17,6 +17,7 @@
 	"rebrand_cities_prefix": "rebrandcitiesdevelopmentsite",
 	"readerFollowingSource": "desktop",
 	"features": {
+		"async-payments": true,
```

Comment:
Just noting that redirect payments are not available in desktop (redirect/popups aren't handled properly) - so while we still want to be able to show the pending payments tab, we don't enable actually paying with a redirect type method in desktop.

Reply:
Cheers, for completeness, we also need to block the checkout regardless of payment type availability.